docs/kyma/docs/04-10-update.md

```diff
@@ -38,13 +37,18 @@ The update procedure consists of three main steps:
   kubectl edit installation kyma-installation
   ```
 
-  > **NOTE:** Read [this](#custom-resource-installation) document to learn more about the Installation custom resource.
+> **NOTE:** Read [this](#custom-resource-installation) document to learn more about the Installation custom resource.
+
+
+- If you introduced changes in overrides, update the existing ConfigMaps and Secrets. Add new ConfigMaps and Secrets if required. See [this](#configuration-helm-overrides-for-kyma-installation) document for more information on overrides.
 
+## Perform the update
 
-- If you introduced changes in overrides, update the existing ConfigMaps and Secrets. Add new ConfigMaps and Secrets if required. See [this](#getting-started-helm-overrides-for-kyma-installation) document for more information on overrides.
+If your changes involve any modifications in the `/resources` folder that includes component chart configurations, you need to perform the whole update process that includes updating the Kyma Installer and triggering the update. If your only modify installation artefacts, for example by adding or removing components in the installation files or adding or removing overrides in the configuration files, you only trigger the update process.
```

Comment:
```suggestion
If your changes involve any modifications in the `/resources` folder that includes component chart configurations, perform the whole update process that includes updating the Kyma Installer and triggering the update. If you only modify installation artifacts, for example by adding or removing components in the installation files or adding or removing overrides in the configuration files, trigger the update process only.
```

Reply:
thanks. I only changed the place of the word "only" based on those guidelines: https://github.com/YaaS/REST_API_Documentation_Guidelines/wiki/009-Only-the-Lonely